test/bridge/Vp8RewriterTest.cpp

```diff
@@ -29,16 +58,31 @@ class Vp8RewriterTest : public ::testing::Test
     std::unique_ptr<bridge::SsrcOutboundContext> _ssrcOutboundContext;
 };
 
-TEST_F(Vp8RewriterTest, getOffset)
+TEST_F(Vp8RewriterTest, ringDifference)
 {
     uint16_t sequenceNumber0 = 22787;
     uint16_t sequenceNumber1 = 2112;
 
-    const int32_t offset1 = utils::Offset::getOffset<int32_t, 16>(sequenceNumber0, sequenceNumber1);
+    const auto offset1 = math::ringDifference<uint16_t, 16u>(sequenceNumber1, sequenceNumber0);
     EXPECT_EQ(20675, offset1);
 
-    const int32_t offset2 = utils::Offset::getOffset<int32_t, 16>(sequenceNumber1, sequenceNumber0);
+    const auto offset2 = math::ringDifference<uint16_t, 16u>(sequenceNumber0, sequenceNumber1);
     EXPECT_EQ(-20675, offset2);
+
+    {
+        const auto offset = math::ringDifference<uint32_t, 12u>(0xFFF, 444);
+        EXPECT_EQ(offset, 445);
+    }
+
+    {
+        const auto offset = math::ringDifference<uint16_t, 16u>(0xFFFF, 444);
+        EXPECT_EQ(offset, 445);
+    }
```

Comment:
Consider reinforcing tests for conventional "rings" (e.g. 16 bits) by checking that, say,  offset + 444 == 0xffff.